src/oclif/CustomCommand.ts

```diff
@@ -1,25 +1,33 @@
-import OclifCommand, { flags as oclifFlags } from '@oclif/command'
-
-import { ParsingToken } from '@oclif/parser/lib/parse'
-
 import { hrTimeToMs } from '../lib/utils/hrTimeToMs'
-import { TelemetryCollector } from '../lib/telemetry/TelemetryCollector'
 import { Metric } from '../lib/metrics/MetricReport'
 import { onError } from './hooks/init'
+import { ParsingToken } from '@oclif/parser/lib/parse'
+import { TelemetryCollector } from '../lib/telemetry/TelemetryCollector'
+import { TraceConfig } from '../lib/globalConfigs/traceConfig';
+import * as Parser from '@oclif/parser';
+import OclifCommand, { flags as oclifFlags } from '@oclif/command'
 
 export abstract class CustomCommand extends OclifCommand {
   public static globalFlags = {
     verbose: oclifFlags.boolean({ char: 'v', description: 'Show debug level logs', default: false }),
     help: oclifFlags.help({ char: 'h' }),
+    trace: oclifFlags.boolean({ char: 't', description: 'Log tracing to jaeger', default: false }),
```

Comment:
We don't have to expose to common users that we use jaeger :) (a advanced user can easily find that info, though :cry: )
```suggestion
    trace: oclifFlags.boolean({ char: 't', description: 'Ensure all HTTP requests to VTEX IO are traced', default: false }),
```